src/methods/sep24/interactiveDepositFlow.ts

```diff
@@ -0,0 +1,55 @@
+import { each } from "lodash";
+import { log } from "helpers/log";
+
+export const interactiveDepositFlow = async ({
+  assetCode,
+  publicKey,
+  sep24TransferServerUrl,
+  token,
+}: {
+  assetCode: string;
+  publicKey: string;
+  sep24TransferServerUrl: string;
+  token: string;
+}) => {
+  const formData = new FormData();
+  const postDepositParams = {
+    asset_code: assetCode,
+    account: publicKey,
+    lang: "en",
+    claimable_balance_supported: "true",
```

Comment:
We should allow users to specify whether or not they want to test the claimable balance flow vs. the flow where the wallet needs to establish trust.

Reply:
Done in https://github.com/stellar/stellar-demo-wallet/pull/126